app/assets/javascripts/discourse/app/components/reviewable-refresh/flag-reason.gjs

```diff
@@ -0,0 +1,28 @@
+import Component from "@glimmer/component";
+import { gt } from "truth-helpers";
+
+const SCORE_TYPE_TO_CSS_CLASS_MAP = {
+  illegal: "illegal",
+  inappropriate: "inappropriate",
+  needs_approval: "needs-approval",
+  off_topic: "off-topic",
+  spam: "spam",
+};
```

Comment:
I guess we could consider `#dasherize` from `@ember/string` to remove one moving part.

**Edit:** I see we're doing that in `ReviewableItem` already, so seems like maybe we should here, too?